### ISSUE TITLE
Syslog input - override_source doesn't do anything

### ISSUE THREAD

**Issue by SjonHortensius** [Graylog2/graylog2-server#848]
I have setup a Syslog input for an OpenBSD machine which somehow doesn't specify the source. So I used override_source in the input; but that doesn't have any effect. Fwiw; the messages look like this:

```
facility: local0
level:    Info [6]
message:  pf: 00:00:01.009955 rule 3/0(match): block in on xxxxxx
source:   pf:
```

The input says that "override_source" is configured properly

**Comment by kroepke** [comment]
can confirm on a raw input

**Comment by kroepke** [comment]
LGTM.
One caveat: plugins that implement AbstractCodec.Config right now need to be recompiled to work after this patch.

**Comment by bernd** [comment]
Fixed by merging #857.
